SU-SOE/stanford_soe_helper
# Soe 2236 refactored homepage featured events view

# READY FOR REVIEW

# Summary
- Refactored homepage Featured Events view, changing markup, removing image, and re-styling as needed

# Needed By (Date)
- End of sprint (10/25)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Revert `stanford_soe_helper_event`
3. Verify that the homepage Feature Events block looks like mockup on ticket (InVision) or on related Google Doc (see links below)

# Affected Projects or Products
- stanford_soe_helper
- SOE

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-2236
- https://projects.invisionapp.com/d/main#/console/10423578/254415347/inspect
- https://docs.google.com/document/d/1LCuqHA9ZTsnGlPDHz0hGI1A5GgJu6VMQrxMfJZNyUn8/edit#heading=h.s255r58sqqu3

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
